tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "allowSyntheticDefaultImports": true,
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
+    "noImplicitAny": false,
```

Comment:
Is this statement required ? It is false by default.

Reply:
removed it, it was needed when using a package md2json, but now we don't require it.